include/kframework/buf.md

```diff
@@ -47,8 +47,8 @@ module BUF
 
     rule #buf(SIZE, DATA) => #padToWidth(SIZE, #asByteStack(DATA %Int (2 ^Int (SIZE *Int 8))))
       requires 0 <Int SIZE
-      [concrete]
-    rule #buf(_SIZE, _) => .Bytes [owise, concrete] // SIZE <= 0
+      [concrete, preserves-definedness]
+    rule #buf(_SIZE, _) => .Bytes [owise, concrete, preserves-definedness] // SIZE <= 0
```

Comment:
Why does this need to be marked as preserving definedness? This is a total function, and `.Bytes` is as well: 
![image](https://user-images.githubusercontent.com/12721972/235472898-fb1a4745-7316-4e2f-8701-51f03eb24971.png)

Reply:
That's true, an oversight.
About totality see the other comment/response